skyportal/handlers/api/obj.py

```diff
@@ -0,0 +1,70 @@
+from baselayer.app.access import auth_or_token
+from ..base import BaseHandler
+from ...models import Candidate, DBSession, Filter, Group, Obj, Photometry, Source
+
+
+class ObjHandler(BaseHandler):
+    @auth_or_token
+    def get(self, obj_id=None):
```

Comment:
What this method implements seems to fall somewhere between HEAD and GET; as is, `ObjHandler.get` implies this will return an `Obj` record, which it doesn't. I think if we're going to break any REST API rules here, it should be in the name of the endpoint -- `ObjExistsHandler`, for example, with an associated endpoint of `/api/obj_exists`

Reply:
I think I agree with this. Since breaking rules seems somewhat necessary, I'd like to hear what @stefanv @dannygoldstein @profjsb and @kmshin1397 have to say about it :) Happy to implement whatever we converge on, just need this endpoint for the upcoming K PR that fixes the alert filtering/posting logic.

Reply:
I would agree with @acrellin as well. Having a `/api/obj_exists` route seems much better to me than an `GET /api/objs` that doesn't return an Object.

Reply:
following a discussion with @stefanv, renamed it to `/api/objs/<oid>/short`.

Reply:
Perhaps I missed that conversation but I would love to hear the justification for this proposal. Certainly doesn't feel intuitive to me.

Reply:
@stefanv could you please comment on this? I have no preference here, so whatever you guys think is the right way, I'm happy to implement; just need this endpoint for K asap :)

Reply:
@acrellin I think the main justification was mainly about not breaking REST best practices and avoiding verbs in the route. Personally, I would agree that `/api/obj_exists` is slightly more intuitive than `/api/objs/<oid>/short` but don't feel strongly enough one way or the other either.

Reply:
To me what makes the most intuitive sense is to have 3 separate handlers - Obj, Source, and Candidate, and each one returns their corresponding mapped record. in this case Obj would return a serialized Obj and fulfill the needs of this endpoint.

Reply:
@dannygoldstein that'd work if we instead implement HEAD handlers for each of those; implementing Obj.get is already too heavy-weight for what's needed here. This is solely for checking the existence of a resource, not for fetching a data representation of it.

Reply:
(that's exactly what HEAD requests are for...)

Reply:
@acrellin Yeah, `ObjHandler`, `SourceHandler`, and `CandidateHandler`, with `get` `head`, `post`, `put` and `delete` would be best.

Reply:
From the REST point of view I agree with you folks, but from a practical standpoint I would need to make potentially 3x api calls to the "`head`s".

Reply:
Implemented the "head"s